controllers/apply/lib/field-types/file.js

```diff
@@ -30,23 +31,17 @@ class FileField extends Field {
             props.attributes
         );
 
-        const schema =
-            props.schema ||
-            Joi.object({
-                filename: Joi.string().required(),
-                size: Joi.number()
-                    .max(maxFileSize.value)
-                    .required(),
-                type: Joi.string()
-                    .valid(supportedMimeTypes)
-                    .required()
-            });
+        this.isRequired = true;
 
-        if (this.isRequired) {
-            this.schema = schema.required();
-        } else {
-            this.schema = schema.optional();
-        }
+        this.schema = Joi.object({
```

Comment:
The file schema shouldn't be overridable as it wouldn't make sense with an alternative schema.